packages/swingset-liveslots/src/liveslots.js

```diff
@@ -257,18 +257,37 @@ function build(
       // eslint-disable-next-line no-await-in-loop, @jessie.js/no-nested-await
       await gcTools.gcAndFinalize();
 
-      // `deadSet` is the subset of those vrefs which lack an in-memory
-      // manifestation *right now* (i.e. the non-resurrected ones), for which
-      // we must check the remaining pillars.
+      // possiblyDeadSet contains a baseref for everything (Presences,
+      // Remotables, Representatives) that might have lost a
+      // pillar. The object might still be supported by other pillars,
+      // and the lost pillar might have been reinstanted by the time
+      // we get here. The first step is to filter this down to a list
+      // of definitely dead baserefs.
```

Comment:
"definitely dead baserefs"
Somehow need to insert a Princess Bride quote from Miracle Max here.